images/win/Windows2016-Readme.md

```diff
@@ -205,7 +205,7 @@ Note: MSYS2 is pre-installed on image but not added to PATH.
 
 ### Database tools
 - Azure CosmosDb Emulator 2.11.13.0
-- DacFx 15.0.4897.1
+- DacFx 15.0.5164.1
```

Comment:
Please don't update auto generating documentation file.

Reply:
ah, apologies

Reply:
@dzsquared, Pls, remove any changes related to *.md files from the PR.

Reply:
@al-cheb finally had a chance to clean up that small mess